backend/api_v2/management/commands/get_org_rate_limit.py

```
@@ -0,0 +1,96 @@
+class Command(BaseCommand):
+    help = "View organization rate limit information and current usage"
+
+    def add_arguments(self, parser):
+        parser.add_argument("org_id", type=str, help="Organization ID or name")
+        parser.add_argument(
+            "--clear-cache",
+            action="store_true",
+            help="Clear cache and force refresh from DB",
+        )
+
+    def handle(self, *args, **options):
```

Ruff (0.14.3)

Unused method argument: `args`

(ARG002)